rust/diesel/src/order_api.rs

```diff
@@ -0,0 +1,35 @@
+use rocket_contrib::json::Json;
+
+use crate::db;
+use crate::order::{NewUserOrder, Order};
+
+#[post("/orders", data = "<order>", format = "json")]
+pub fn create_order(order: Json<NewUserOrder>, connection: db::Connection) -> &'static str {
+    Order::create(order.into_inner(), &connection);
+    "TBD"
+}
+
+#[get("/orders")]
+pub fn read_orders(connection: db::Connection) -> Json<Vec<Order>> {
```

Comment:
Its a bit inconsistent to return a `Vec` here, since in other places you are wrapping lists in a separate type with a `content` field. I don't see anything about the desired output of this api in the readme, so I would guess it is flexible, but I may be missing something.

Reply:
Fixed